.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: ci
+on: [push]
+jobs:
```

Comment:
Do you think it's worth running the tests across multiple Node.js versions, like we previously did with Travis CI? Here's an example for doing so: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs#specifying-the-nodejs-version

Reply:
Not really. I dislike wasting computer cycles, even if they aren't on my own computers.